src/modules/ipfix/IpfixSender.cpp

```diff
@@ -216,6 +217,7 @@ void IpfixSender::addCollector(const char *ip, uint16_t port,
 	    case TCP:
 	        msg(MSG_INFO, "%sIpfixSender: adding TCP://%s:%d to exporter",
 				vrf_log, ip, port);
+			__FALLTHROUGH__;
```

Comment:
Should this really be fallthrough? Wouldnt this mean that we throw an exception for TCP export?

Reply:
I assumed so, based on the description of the ipfix_add_collector function, which does not include TCP in the list of accepted protocols, and the description of the enum ipfix_transport_protocol which states: "TCP, currently unsupported by ipfixlolib".
Perhaps we should fix this at some point.

Reply:
👍